README.md

```diff
@@ -76,6 +77,37 @@ $obj.Rename("OtherGuest")
 
 This has the added benefit of not requiring escaping '$' in the PowerShell code. Note that the files need to have DOS linefeeds or they will not work as expected. One tool for converting UNIX linefeeds to DOS linefeeds is [unix2dos](http://freecode.com/projects/dos2unix).
 
+### External files and exit codes
+If you are calling external files, such as other PowerShell scripts or executables, be aware that the last executed script's exitcode will be used by Puppet to determine whether the command was succesful.  For example:
+
+Suppose the file `C:\fail.ps1` contains the following PowerShell script
+
+~~~ powershell
```

Comment:
Without getting too carried away, I think we should have a multi-line script sample that calls an external executable multiple times that clearly demonstrates the last command being executed as the overriding exit code value.